engine/CMakeLists.txt

```diff
@@ -1407,7 +1409,7 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
 ELSE ()
     MESSAGE("-- Configuring Packaging for Unknown Platforms - \"${CMAKE_SYSTEM_NAME}\"")
     # Unknown Platform --> Just do the various compressed tarballs
-    SET(CPACK_GENERATOR "TBZ2" "TGZ" "TXZ")
+    SET(CPACK_GENERATOR "TXZ")
```

Comment:
Not blocking as it doesn't affect CI: This can really be any/all of them.
It doesn't affect CI as we don't do a build on an unknown platform; so this would really only apply for local builds at the moment.